pcdsdevices/usb_encoder.py

```diff
@@ -0,0 +1,64 @@
+"""
+Class to handle USB encoders
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal
+
+from .interface import BaseInterface
+
+
+class USBEncoder(BaseInterface, Device):
+    """
+    Class for the USB encoder by US Digital.
+    https://www.usdigital.com/products/accessories/interfaces/usb/usb4
+
+    Parameters
+    ----------
+    prefix: str
+        Base PV of the USB encoder
+
+    name: str
+        USB encoder name
+
+    linked_axis: device
+        Instance of the axis on which the encoder is mounted
+    """
+    zero = Cpt(EpicsSignal, ':ZEROCNT', kind='omitted')
+    pos = Cpt(EpicsSignal, ':POSITION', kind='hinted')
+    scale = Cpt(EpicsSignal, ':SCALE', kind='config')
+    offset = Cpt(EpicsSignal, ':OFFSET', kind='config')
+
+    tab_whitelist = ['pos', 'set_zero', 'set_zero_with_axis',
+                     'scale', 'offset']
+
+    def __init__(self, prefix, *, name, linked_axis=None, **kwargs):
+        super().__init__(prefix, name=name, **kwargs)
+        self.linked_axis = linked_axis
+
+    def set_zero(self):
+        """ Resets the encoder counts to 0 """
+        self.zero.put(1)
+
+    def set_zero_with_axis(self):
+        """
+        Resets the encoder counts to 0 and sets the stage offset
+        so that its position is also 0.
+        """
+        self.zero.put(1)
+        if (
+            self.stage is not None or
```

Comment:
This will always evaluate to `True` because `stage` is a method inherited from `Device`

Reply:
Good catch, fixing it in the next commit